awscli/examples/scheduler/create-schedule-group.rst

```diff
@@ -0,0 +1,14 @@
+**To create a schedule group**
+
+The following ``create-schedule-group`` example creates a schedule group. ::
+
+    aws scheduler create-schedule-group \
+        --name DailySNSFunction
+
+Ouput ::
```

Comment:
```suggestion
Output::
```
This line needs to be fixed in all your examples.

Reply:
made changes to all examples. published new commit